### PR TITLE
Contents page update

### PR DESCRIPTION
**Pull request by PexMor**
Could we update the Contents page to better reflect the directory names and add the missing 6th and 7th session?

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -11,21 +11,46 @@ community.
...
+
+*** Operator SDK
+
+[[./06_operator_sdk][06_operator_sdk]]
+
+*** Ansible Operator
+
+[[./07_ansible_operator][07_ansible_operator]]
+
+*** Ansible Operator for K8S
```
Comment (jwerak) [inline README.org]: Duplicate?

Comment (PexMor) [reply README.org]: Well the intention is to have visible directory names as well as human friendly H3 label, some short description of the contents shall follow -> To-Do

Comment (PexMor) [reply README.org]: in case you meant the e.g. ```[[./07_ansible_operator][07_ansible_operator]]```

Comment (jwerak) [reply README.org]: nope, I meant 
`*** Ansible Operator for K8S` 

It doesn't have any link and seems like just another version of `*** Ansible Operator`

Comment (PexMor) [reply README.org]: Yes, you were right I just missed the line. Fixed now.

**Comment by PexMor** [comment]
Could get its way to merge? Or any suggestions about the content?

**Pull request merged**